app/src/main/java/com/github/mobile/gauges/ui/ListLoadingFragment.java

```diff
@@ -74,8 +84,19 @@ public boolean onOptionsItemSelected(MenuItem item) {
      * Refresh the fragment's list
      */
     public void refresh() {
-        if (getActivity() != null)
+        if (getActivity() != null) {
+            /* Attach a rotating ImageView to the refresh item as an ActionView */
+            LayoutInflater inflater = (LayoutInflater) getActivity().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+            ImageView iv = (ImageView) inflater.inflate(layout.refresh_action_view, null);
+
+            Animation rotation = AnimationUtils.loadAnimation(getActivity(), anim.clockwise_refresh);
+            rotation.setRepeatCount(Animation.INFINITE);
```

Comment:
i think you can remove this line and just add the following to your `refresh.xml`:

        android:repeatCount="infinite"
        android:repeatMode="restart"